js/webrtc.js

```diff
@@ -931,6 +931,13 @@ var spreedPeerConnectionTable = [];
 			if (!OCA.SpreedMe.app.videoDisabled) {
 				OCA.SpreedMe.app.enableVideo();
 			}
+
+			var $hideVideoButton = $('#hideVideo');
+			if(OCA.SpreedMe.webrtc.webrtc.localStream.getVideoTracks() < 1) {
+				$hideVideoButton.removeClass('video-disabled icon-video-off-white')
+				.addClass('no-video-available icon-video-off-white')
```

Comment:
You remove `icon-video-off-white` and add it immediately back, this is unnecessary.